javascript1/week4/preparation.md

```diff
@@ -7,3 +7,20 @@ Go through the learning goals of week 1, week 2 and week 3 of javascript. Make s
 - [Week 3 learning goals](../week3/readme.md)
 
 _Please go through the material and come to class prepared!_
+
+## Soft Skills of a Software Engineer
+When you start applying for jobs, you will notice a significant focus on hard skills. However, this does not diminish the importance of soft skills. In Denmark, there is generally a greater emphasis on soft skills compared to many other countries.
+
+### Important Soft Skills Early in Your Career
+Some of the most important soft skills at the beginning of your career are:
+- **Problem-Solving**
+- **Communication & Collaboration**
+- **Continuous Learning & Adaptability**
+
+#### Why Denmark Values Soft Skills Highly
+- **Flat Organizational Structures**: Danish companies often have a flat structure, where developers are expected to take responsibility, communicate openly, and work closely with colleagues and managers.
+- **Agile Methodologies**: Many Danish tech companies follow Scrum and other agile methodologies, where teamwork and communication are key to success.
+- **Ownership and Participation**: Developers in Denmark are expected to take ownership of their tasks and actively participate in decision-making processes.
+- **Consensus Culture**: Denmark has a strong consensus culture, meaning that decisions are often made through discussion and mutual agreement. This requires good communication skills, collaboration, and the ability to consider different perspectives.
+
```

Comment:
It'd be nice to add some links on this topics. I've read about it on forums but maybe there are some more "official" ones.

Reply:
+1, a link or two to describe each of these in more detail would be nice for more context. It might be the first time people hear about e.g. consensus culture or agile/scrum.